project3/styles.css

```diff
@@ -0,0 +1,285 @@
+* {
+    box-sizing: border-box;
+    transition: 400ms;
+}
+
+
+body {
+    background-color: #163346;
+    margin: auto;
+
+}
+
+div {
+    background-color: #1e2019;
+    margin: auto;
+    max-width: 70%;
+    padding: 1.25%;
+
+
+}
+
+fieldset {
+    border: 0;
+}
+
+fieldset span {
+    display: block;
+}
+
+
+
+
+header {
+    background-color: #D1D5DE;
+
+
+}
+
+h1 {
+
+    color: #000000;
+    font-family: 'Teko', sans-serif;
+    font-size: 48px;
+    font-weight: 700;
+    text-transform: uppercase;
+
+}
+
+h2 {
+    color: #fff;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    font-weight: 500;
+    text-decoration: underline;
+    text-decoration-color: #163346;
+    text-transform: uppercase;
+}
+
+input[type="radio"] {
```

Comment:
I would recommend looking to combine the radio and checkbox styles that are similar to avoid having to repeat so much code, e.g:

```
input[type="radio"], input[type="checkbox"] {
    -moz-appearance: none;
    -webkit-appearance: none;
    appearance: none;
    border: 3px solid #eaecf0;
    cursor: pointer;
    height: 24px;
    width: 24px;
}
input[type="radio"] {
    border-radius: 50%;
}
```